src/codeu/chat/server/ViewDatabase.java

```diff
@@ -0,0 +1,371 @@
+                    found.add(user);
+                }
+                rs.close();
+                stmt.close();
+            }
+
+            connection.close();
+        } catch ( Exception e ) {
+            System.err.println( e.getClass().getName() + ": " + e.getMessage() );
+            System.exit(0);
+        }
+
+        return found;
+    }
+
+    public static Collection<Conversation> getConversations(Collection<Uuid> ids){
+        final Collection<Conversation> found = new HashSet<>();
+
+        System.out.println("Accessing ViewDatabase");
+
+        Connection connection = null;
+        Statement stmt = null;
+
+        try {
+            Class.forName("org.sqlite.JDBC");
+            connection = DriverManager.getConnection("jdbc:sqlite:./bin/codeu/chat/codeU_db/ChatDatabase.db");
+            connection.setAutoCommit(false);
+
+            for(final Uuid id : ids) {
+                stmt = connection.createStatement();
+                ResultSet rs = stmt.executeQuery( "SELECT * " +
+                        "FROM CONVERSATIONS " +
+                        "WHERE  ID = "+SQLFormatter.sqlID(id)+";" );
+                while (rs.next()){
+                    Uuid conversationID = Uuid.parse(rs.getString("ID"));
+                    String conversationName = rs.getString("CNAME");
+                    Time creationTime = Time.fromMs(rs.getLong("TimeCreated"));
+                    Uuid ownerID = Uuid.parse(rs.getString("OWNERID"));
+
+
+                    Conversation conversation = new Conversation(conversationID, ownerID, creationTime, conversationName);
+                    found.add(conversation);
+                }
+                rs.close();
+                stmt.close();
+            }
+
+            connection.close();
+        } catch ( Exception e ) {
+            System.err.println( e.getClass().getName() + ": " + e.getMessage() );
+            System.exit(0);
+        }
+
+        return found;
+    }
+
+    public static Collection<Message> getMessages(Collection<Uuid> ids){
+        final Collection<Message> found = new HashSet<>();
+
+        System.out.println("Accessing ViewDatabase");
+
+        Connection connection = null;
+        Statement stmt = null;
+
+        try {
+            Class.forName("org.sqlite.JDBC");
+            connection = DriverManager.getConnection("jdbc:sqlite:./bin/codeu/chat/codeU_db/ChatDatabase.db");
+            connection.setAutoCommit(false);
+
+            for(final Uuid id : ids) {
+                stmt = connection.createStatement();
+                ResultSet rs = stmt.executeQuery( "SELECT * " +
+                        "FROM MESSAGES " +
+                        "WHERE ID = "+SQLFormatter.sqlID(id)+";" );
+                while (rs.next()){
+                    Uuid messageID = Uuid.parse(rs.getString("ID"));
+                    Uuid nextMessageID = Uuid.parse(rs.getString("MNEXTID"));
```

Comment:
It might be a good idea to put field names in static final constants in the DB helper class so that the field names used here can stay in sync with whatever names are used to create the tables.